Doc/library/itertools.rst

```diff
@@ -795,12 +795,12 @@ on the Python Package Index::
     python -m pip install more-itertools
 
 Many of the recipes offer the same high performance as the underlying toolset.
-Superior memory performance is kept by processing elements one at a time
-rather than bringing the whole iterable into memory all at once. Code volume is
-kept small by linking the tools together in a functional style which helps
-eliminate temporary variables.  High speed is retained by preferring
-"vectorized" building blocks over the use of for-loops and :term:`generator`\s
-which incur interpreter overhead.
+Superior memory performance is kept by processing elements one at a time rather
+than bringing the whole iterable into memory all at once. Code volume is kept
+small by linking the tools together in a `functional style
+<https://www.cs.kent.ac.uk/people/staff/dat/miranda/whyfp90.pdf>`_.  High speed
+is retained by preferring "vectorized" building blocks over the use of for-loops
+and :term:`generator`\s which incur interpreter overhead.
```

Comment:
This renders a bit more nicely:
```suggestion
and :term:`generators <generator>` which incur interpreter overhead.
```

<table>
<tr>
<th>Before
<th>After
<tr>
<td>
<img width="811" alt="image" src="https://github.com/python/cpython/assets/1324225/e3a6af2f-51ed-4bd2-b642-3045f70254c3">

<td>
<img width="808" alt="image" src="https://github.com/python/cpython/assets/1324225/a066fe41-cc00-4829-9e3c-162e1077f841">

</table>